heliopy/data/wind.py

```diff
@@ -107,16 +109,66 @@ def swe_h1(starttime, endtime):
                  'Proton_Wpar_moment': 99999.9,
                  'Alpha_Na_nonlin': 100000.0,
                  'Alpha_sigmaNa_nonlin': 100000.0}
-    keys = {'T_a': 'T_a',
-            'T_p': 'T_p',
-            'n_a': 'n_a',
-            'n_p': 'n_p',
-            'va_x': 'va_x',
-            'va_y': 'va_y',
-            'va_z': 'va_z',
-            'vp_x': 'vp_x',
-            'vp_y': 'vp_y',
-            'vp_z': 'vp_z'}
+    keys = {'fit_flag': 'fit_flag',
```

Comment:
I think these bits can be deleted now that all the cdf keys are automatically copied across.

Reply:
Yep! Need to rebase this and also remove some useless bits now, but since the def cdf_units function will make some more changes, so I'm going to rebase it after that one.

Reply:
👍